app/controllers/steps_controller.rb

```diff
@@ -14,6 +14,22 @@ def destroy_all
     head :no_content
   end
 
+  # endpoint only used for ocrWF to skip all steps if files don't exist
+  # Done to make sure we don't have to load the entire cocina in Argo
+  # Added as endpoint to avoid making multiple calls to update_status for all steps in WF
```

Comment:
```suggestion
Endpoint currently only used for ocrWF to skip all steps if files don't exist (as of June 2024)
Added as endpoint to centralize logic for future use and to allow all updates to be made in one transaction
```

Reply:
In fact, now that I just typef it, I'm not even sure I like my own suggestion :)

It's prone to be wrong if and when we use this endpoint somewhere else in the future and don't update the comment to match.

Perhaps even just something like `Used by ocrWF to skip all steps if files don't exist` or something that is a pointer without being super specific about if anyone else is also using it

Reply:
oh, fair point, i like the idea of making the comment a little less specific so that it's less prone to going stale.

Reply:
and just as i type that, i see an update to the comment 🙂 (updated version lgtm)